api/operator.go

```diff
@@ -79,3 +99,61 @@ func (op *Operator) RaftRemovePeerByAddress(address string, q *WriteOptions) err
 	resp.Body.Close()
 	return nil
 }
+
+// KeyringInstall is used to install a new gossip encryption key into the cluster
+func (op *Operator) KeyringInstall(key string) error {
```

Comment:
No, but I'd `q *QueryOptions` and `q *WriteOptions`arguments to these methods and get the token from there. It kind of sucks that there's some extra fields in those that don't matter here, but that's kind of the common signature for these. We could add a helper that errors if there are things that are not supported, but we can just document for now.